test/files/neg/nested-class-shadowing.check

```diff
@@ -1,4 +1,4 @@
-nested-class-shadowing.scala:9: warning: class shadowing is deprecated but class Status shadows class Status defined in trait Core; rename the class to something else
+nested-class-shadowing.scala:9: warning: nested class shadowing is deprecated but class Status shadows class Status defined in trait Core; rename the class to something else
```

Comment:
warning: class shadowing that looks like overriding is deprecated but class Status shadows class Status defined in trait Core; rename the class to something else

?

Reply:
warning: shadowing a nested class of a parent class is deprecated but class Status shadows class Status defined in trait Core; rename the class to something else

?

Reply:
warning: shadowing an inherited nested class is deprecated but class Status shadows class Status defined in trait Core; rename the class to something else

?